railties/lib/rails/application.rb

```diff
@@ -220,6 +220,22 @@ def message_verifier(verifier_name)
     #     Rails.application.configure do
     #       config.middleware.use ExceptionNotifier, config_for(:exception_notification)
     #     end
+    #
+    #     # It also merges configurations in a shared section deeply:
```

Comment:
The same change from above should be made here. We should only say "You can also store configurations in a shared section which will be merged with the environment configuration".